lib/module.js

```diff
@@ -17,6 +17,10 @@ module.exports = async function NuxtLoadingScreen () {
     loading.setStates(states)
   })
 
+  this.nuxt.hook('cli:buildError', (error) => {
```

Comment:
Can't we hook into builder/webpack instead?

Reply:
Forget about it! This is much better :D

Reply:
I am happy to change it if you want if you think there is a better way, I just wasnt in a _refactor all the things_ mode and tried to be as less invasive as possible. But fully understand/agree that implementations resulting from such a mode could be better optimized if we would refactor other stuff as well.